### ISSUE TITLE
Deadzone fix no longer works

### ISSUE THREAD

**Issue by aaronson2012** [matzman666/OpenVR-InputEmulator#65]
As of version 1.3 the deadzone fix no longer works. Other trackpad emulation features seem to work however.

**Comment by FolkSong** [comment]
Update: with the SteamVR "openvr-inputemulator-temporary" beta it works correctly, so I suggest everyone stay on that as long as possible.

But if they take the beta away we'll be in trouble.

**Comment by cheddarva** [comment]
I can confirm this as well. I've been trying to get my right hand GRIP to function as the Axis 0 center click. Without the deadzone fix this doesn't work.
Also, FolkSong, they valve stated the temporary beta would only last until the 22nd........ being tomorrow

**Comment by matzman666** [comment]
The deadzone fix still works with the joystick on my DIY rifle controller. Apparently Valve completely changed the way the Touch joysticks are handled. And without owning a Rift I have a hard time finding out how they are handled now.

Using [OpenVR-Inspector](https://github.com/matzman666/OpenVR-Inspector) could anyone try to click on the joystick with the deadzone fix enabled and tell me which x/y-values are reported by Axis0?

> I've been trying to get my right hand GRIP to function as the Axis 0 center click

The grip button on the Touch controller is an analog input, and mapping analog inputs to digital buttons (or vice-versa) is currently not supported. I can add some quick and dirty fixes to get mapping from digital buttons to analog inputs working, but the other way round is more difficult and I won't be able to add it any time soon.

**Comment by Nimbus89** [comment]
As soon as I put on the headset, all axis values become "NA". As soon as I take it off, it goes back to being correct values. When I have the headset off and the deadzone fix applied, it reports 0.01 on x and y. I've got some experience with QT etc, so if you need some debugging done with actual touch controllers, give me a shout.

**Comment by hookmanuk** [comment]
Just tried the OpenVR-Inspector as suggested.

As Nimbus89 said, with the headset on all values are NA.

Without the headset, with or without the deadzone fix applied I get x=0 & y=0 whilst I press the right thumbstick. As expected, the deadzone fix doesn't work in game for me either.

**Comment by Nimbus89** [comment]
Ooh, scratch what I said about the values being 0.01. Forgot I was still in the steam VR beta and using 1.2.

**Comment by gregtampa** [comment]
I can activate stuff by pressing in on the thumbstick. But the stick can't be in middle so I like push it up and then in to interact with things.
It's a pain but eh...

**Comment by mhook-winman** [comment]
Yep, presumably from comments on here the fix used to set the X,Y values to 0.01 when you click the joystick, so that fallout would recognise the click.

This doesn't happen anymore.
